src/app/templates/Dockerfile

```diff
@@ -0,0 +1,69 @@
+FROM ubuntu:14.04.3
+
+MAINTAINER Kyle Campbell <kc@kc.io>
+
+ENV TERM xterm
+
+RUN useradd -m docker && echo "docker:docker" | chpasswd && adduser docker sudo
+
+RUN apt-get update
+RUN apt-get -y install sudo
+
+RUN apt-get install -y build-essential \
```

Comment:
Nah, in the long run its way better to build from as close to scratch as you can. May rip out ubuntu under the hood to make for a lighter distro, but this is a proven one for me so I'm pretty partial to it even if it is a bit heavy.